OpenRA.Game/FieldLoader.cs

```diff
@@ -620,12 +634,17 @@ public static object GetValue(string fieldName, Type fieldType, string value)
 
 		public static object GetValue(string fieldName, Type fieldType, string value, MemberInfo field)
 		{
-			return GetValue(fieldName, fieldType, new MiniYaml(value), field);
+			return GetValue(fieldName, fieldType, value, null, field);
```

Comment:
Are we sure passing `null` here won't cause a crash when we run into the `ParseDictionary` parser?

Reply:
Did some testing and in fact this is possible and can cause a crash. Nicely spotted.

Added the following to `ParseDictionary` so it creates an empty dict as it would have previously.
```c#
if (yaml == null)
	return Activator.CreateInstance(fieldType);
```